pisek/config/task_config.py

```diff
@@ -701,6 +701,58 @@ def load_dict(
     def validate_name(cls, value: str) -> str:
         return _validate_program_name("program name", value)
 
+    @field_validator("env", mode="before")
+    def interpolate_env(cls, dictionary: dict[str, str]) -> dict[str, str]:
+        def raise_err(error_type: str, message: str, key: str):
+            raise PydanticCustomError(
+                error_type, message, {"_loc": key, f"env_{key}": dictionary[key]}
+            )
+
+        def interpolate(key: str, val: str) -> str:
+            result: list[list[str]] = [["normal", ""]]
```

Comment:
Is a parser state really a homogenous sequence of strings of variable length?

If this is the result, why isn't it returned?